bin/cron/sync_dropbox

```diff
@@ -33,26 +34,35 @@ FOLDERS = {
 
 INTERVAL_SECONDS = 5
 TOTAL_SECONDS = 60 - INTERVAL_SECONDS
-error_message = ""
-alert_slack = true
+
+logger = HighFrequencyReporter.new(Slack, 'sync-dropbox-staging', '/home/ubuntu/dropbox_sync_error_log.csv')
+logger.load # load errors from most recent sync
+
 while (Time.now - SCRIPT_START) < TOTAL_SECONDS
   attempt_start = Time.now
+  logger.reset_new_events # reset new events such that they represent one run through all FOLDERS
   FOLDERS.each do |key, value|
     # 'unison' will sync the two folders. It will return true on success and false on failure.
     # For full details, see unison's man page. Docs are also here: https://www.cis.upenn.edu/~bcpierce/unison/download/releases/stable/unison-manual.html
     command = "unison /home/ubuntu/Dropbox/Shared/#{key} /home/ubuntu/staging/pegasus/#{value} -silent -ignore \"Name .dropbox\" -auto -perms 0 -dontchmod"
     stdout, stderr, _ = Open3.capture3(command)
-    if stdout != "" || stderr != ""
-      error_message = "#{error_message}Dropbox directory: #{key}. staging directory: #{value}. \n#{stdout}#{stderr}\n"
-    end
+    next if stdout == "" && stderr == ""
+    error_message = <<~ERROR_MSG
+                      <!here> Error syncing Dropbox and staging
+                      Dropbox directory: #{key}
+                      pegasus directory: #{value}
+                      stdout: #{stdout}
+                      stderr: #{stderr}
+                    ERROR_MSG
```

Comment:
I think this error message is fine. If you wanted to get fancier you could only show stdout/stderr if it was not empty, but I don't think that's necessary

Reply:
There is value in showing both stdout & stderr. When the error messages are not super informative, seeing both outputs helps eliminating the doubt that there might be more info elsewhere.